07_train_final_classifiers/collect_full_models.ipynb

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.            params[gene_name] = pd.DataFrame(
</pre><p>Minor suggestion (if I understand correctly): seems that this could be a <code>pandas.Series</code>, where its <code>name</code> is <code>gene_name</code>.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>